src/tasks/generate-storage-layout.ts

```diff
@@ -65,6 +65,11 @@ const preprocessTable = (tableContent: string) => {
   return listItemOfTable;
 };
 
+function removeIdentifierSuffix(type: string) {
+  const identifierRegex = /\d+_(storage|memory|calldata)/g; // 12_memory 1123_storage
+  return type.replace(identifierRegex, '');
```

Comment:
should keep the suffix identifier, but remove the id

idea:
```suggestion
  const identifierRegex = /(\d+)_(storage|memory|calldata)/g; // 12_memory 1123_storage
  return type.replace(identifierRegex, 'static_$2');
```

Reply:
https://stackoverflow.com/a/3598167